libafl/src/executors/inprocess.rs

```diff
@@ -1172,7 +1170,14 @@ mod windows_exception_handler {
                     .0,
             )
             .unwrap();
-            eprintln!("Crashed with {code}");
+
+            let exception_list = data.exceptions();
+            if exception_list.contains(&code) {
+                eprintln!("Crashed with {code}");
+            } else {
+                eprintln!("Exception code received, but {code} is not in CRASH_EXCEPTIONS");
```

Comment:
Would it be good to change the crashing signals at runtime? So, V8 needs to handle sigtrap on linux